lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1257,6 +1257,9 @@ def has_different_configuration(self, image):
             volume_driver=host_config.get('VolumeDriver')
         )
 
+        if kwargs.get('ignore_image', False):
```

Comment:
Thank you for pointing this out, actually we can get `ignore_image` from `self.parameters`.

I will update this soon.